importer/api/__init__.py

```diff
@@ -144,6 +144,19 @@ def import_course(bundle, repo_id, user_id, static_dir):
     return course
 
 
+def is_leaf_tag(tag):
+    """
+    Should we look for resources within elements with this tag?
+
+    Args:
+        tag (unicode): Element tag
+    Returns:
+        bool: Whether tag is leaf tag
+    """
+    return tag in ('video', 'html', 'problem', 'discussion')
```

Comment:
Yeah, a set literal is great. It doesn't have to be `set()` -- I was just referring to a set.

And yeah, it's only four elements. That's why I said "ultra-nit." 😉